### ISSUE TITLE
Improvement: two levels for visible a11y violations

### ISSUE THREAD

**Issue by MelSumner** [ember-a11y/ember-a11y-testing#34]
Recommend two settings for visible a11y violations, with a more subdued error look by default. (maybe keep the stripes but make them grayscale?) 

Reasoning is that when testing in a business environment, noted strong negative reactions to how the errors came across visually. Don't want to have anyone avoid either this addon or the issue of accessibility merely because the validation addon seems a bit strong, but do think the option should be there for environments where the stronger the better!

**Comment by BrianSipple** [comment]
I like it. I'm getting after this now, and I'm getting the sense that if we're going to offer this functionality, three levels would actually be even better. 

I'm working on an API concept of "noise levels" -- which right now are just `1`, `2`, and `3`, but could also be keywords like `soft`, `average`, and `loud`.

**Comment by MelSumner** [comment]
Closing this issue since we've already added noise levels.
